### ISSUE TITLE
[Article Proposal]  Transfer Learning for Image Classification Tutorial

### ISSUE THREAD

**Issue by ayyucedemirbas** [kili-technology/kili-blogger-program#26]
### My resource

**Topic:**

Transfer Learning for Image Classification Tutorial

**Outline:**

In this tutorial, you will dive into deep learning model training basics such as data preprocessing, how gradient descent works, gradient descent optimization algorithms, batch normalization, how convolutional neural nets work, and the structures of widely used pre-trained models. After this background, you will learn the basics of image classification and transfer learning while training your own model. 

### My content is

- [x] A Kili Tutorial / Guide / How to article
- [ ] An Article

**Comment by theodu** [comment]
Hello @ayyucedemirbas 
This content is too model-centric. Our content should be focused on data and how to optimize model performances while iterating on data. We won't select this article proposal

Thank you and best wishes!
